atom-publisher-lib/src/test/scala/com/gu/atom/data/DynamoDataStoreSpec.scala

```diff
@@ -23,35 +23,47 @@ class DynamoDataStoreSpec
     with BeforeAndAfterAll
     with AtomImplicitsGeneral {
   val tableName = "atom-test-table"
+  val publishedTableName = "published-atom-test-table"
 
-  type FixtureParam = DynamoDataStore[MediaAtom]
+  type FixtureParam = (PreviewDynamoDataStore[MediaAtom], PublishedDynamoDataStore[MediaAtom])
```

Comment:
could we make this a case class with some named members, just to make the code a bit more readable (to avoid referencing `_1` etc)

Something like:

```scala
case class DataStores(preview: PreviewDynamoDataStore[MediaAtom],
  published: PublishedDynamoDataStore[MediaAtom])
type FixtureParam = DataStores
```